.travis.yml

```diff
@@ -4,7 +4,7 @@ go:
 install:
  - go get github.com/golang/lint/golint
  - go get github.com/fzipp/gocyclo
- - go get github.com/sirupsen/logrus
```

Comment:
What's with the capitalisation?

Reply:
well, good question.

It appears that this package is referred to as `github.com/Sirupsen/logrus` almost everywhere else. The inconsistent naming appears relatively harmless, except that we end up with two copies: https://github.com/matrix-org/dendrite/tree/master/vendor/src/github.com/sirupsen/logrus, https://github.com/matrix-org/dendrite/tree/master/vendor/src/github.com/Sirupsen/logrus.

Anyway, since some part of the go tooling had fixed this for me, I thought I might as well include it here.